From d509e486fa2a931c177292ce0cb81ceecd24c81d Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Fri, 13 Oct 2023 08:06:52 +0000
Subject: Use correct install locations

Use the correct install locations for QtAsyncio
and designer plugin.

Pick-to: 6.6
Fixes: PYSIDE-2494
Change-Id: I760e8f52f19cd1f1cdf742a15df071c4f4deffa5
Reviewed-by: Qt CI Bot <qt_ci_bot@qt-project.org>
Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>
---
 build_scripts/platforms/unix.py                 | 2 +-
 build_scripts/platforms/windows_desktop.py      | 2 +-
 sources/pyside6/PySide6/CMakeLists.txt          | 2 +-
 sources/pyside6/plugins/designer/CMakeLists.txt | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/build_scripts/platforms/unix.py b/build_scripts/platforms/unix.py
index 5ba41fe25..6b4a3e95d 100644
--- a/build_scripts/platforms/unix.py
+++ b/build_scripts/platforms/unix.py
@@ -212,7 +212,7 @@ def prepare_packages_posix(pyside_build, _vars, cross_build=False):
         # <source>/pyside6/{st_package_name}/QtAsyncio/* ->
         #   <setup>/{st_package_name}/QtAsyncio/*
         copydir(
-            f"{{build_dir}}/{PYSIDE}/{{st_package_name}}/QtAsyncio",
+            f"{{site_packages_dir}}/{{st_package_name}}/QtAsyncio",
             "{st_build_dir}/{st_package_name}/QtAsyncio",
             _vars=_vars)
 
diff --git a/build_scripts/platforms/windows_desktop.py b/build_scripts/platforms/windows_desktop.py
index f8ad4c8c8..543ee9880 100644
--- a/build_scripts/platforms/windows_desktop.py
+++ b/build_scripts/platforms/windows_desktop.py
@@ -166,7 +166,7 @@ def prepare_packages_win32(pyside_build, _vars):
         # <source>/pyside6/{st_package_name}/QtAsyncio/* ->
         #   <setup>/{st_package_name}/QtAsyncio/*
         copydir(
-            f"{{build_dir}}/{PYSIDE}/{{st_package_name}}/QtAsyncio",
+            f"{{site_packages_dir}}/{{st_package_name}}/QtAsyncio",
             "{st_build_dir}/{st_package_name}/QtAsyncio",
             _vars=_vars)
 
diff --git a/sources/pyside6/PySide6/CMakeLists.txt b/sources/pyside6/PySide6/CMakeLists.txt
index e24fdb682..37b7a6c9c 100644
--- a/sources/pyside6/PySide6/CMakeLists.txt
+++ b/sources/pyside6/PySide6/CMakeLists.txt
@@ -111,4 +111,4 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/pyside6_global.h
         DESTINATION include/${BINDING_NAME}${pyside6_SUFFIX})
 
 install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/QtAsyncio"
-        DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
+        DESTINATION "${PYTHON_SITE_PACKAGES}/${BINDING_NAME}${pyside6_SUFFIX}")
diff --git a/sources/pyside6/plugins/designer/CMakeLists.txt b/sources/pyside6/plugins/designer/CMakeLists.txt
index 570fee1fe..c86d328cf 100644
--- a/sources/pyside6/plugins/designer/CMakeLists.txt
+++ b/sources/pyside6/plugins/designer/CMakeLists.txt
@@ -53,4 +53,4 @@ target_link_libraries(PySidePlugin PRIVATE
                       Qt::Widgets
                       ${SHIBOKEN_PYTHON_LIBRARIES})
 
-install(TARGETS PySidePlugin LIBRARY DESTINATION "plugins/designer")
+install(TARGETS PySidePlugin LIBRARY DESTINATION "${QT6_INSTALL_PLUGINS}/designer")
-- 
cgit v1.2.3

